split.py

```diff
@@ -266,6 +268,9 @@ def main(config_path, modes, verbose, use_cache=True):
     with options.opts.target_path.open("rb") as f2:
         rom_bytes = f2.read()
 
+    if options.opts.platform == "gc":  # TODO move this into platform initialization
```

Comment:
isn't platform initialization implemented?

Reply:
Indeed, I moved the function to the GC platform init function.